src/Libraries/Microsoft.Extensions.AI.Evaluation.Console/Program.cs

```diff
@@ -105,5 +139,16 @@ private static async Task<int> Main(string[] args)
 #endif
 
         return await rootCmd.InvokeAsync(args).ConfigureAwait(false);
+
+        void requiresPathOrEndpoint(CommandResult cmd)
+        {
+            bool hasPath = cmd.GetValueForOption(pathOpt) is not null;
+            bool hasEndpoint = cmd.GetValueForOption(endpointOpt) is not null;
+            if (!(hasPath ^ hasEndpoint))
```

Comment:
Consider changing to -

```suggestion
            bool hasPath = string.IsNullOrWhiteSpace(cmd.GetValueForOption(pathOpt));
            bool hasEndpoint = string.IsNullOrWhiteSpace(cmd.GetValueForOption(endpointOpt));
            if (!(hasPath ^ hasEndpoint))
```

Reply:
Ah, they are not string values. They are strongly typed. `DirectoryInfo` and `Uri` respectively.

Reply:
Ah my bad 😄